packages/aws-rfdk/lib/deadline/lib/worker-fleet.ts

```diff
@@ -182,13 +183,17 @@ export interface WorkerInstanceFleetProps {
    * Health Monitor component to monitor the health of instances.
    *
    * @default - Health Monitoring is turned-off
+   *
+   * Note: The health-check feature is supported with Deadline Client v10.1.9 and later.
```

Comment:
I'm not 100% positive, but it is possible this note might get parsed as part of the `@default` TSDoc block. Here's what I know:

Unfortunately, TSDoc has no official spec yet, but the [TSDoc GitHub README](https://github.com/microsoft/tsdoc#why-do-we-need-tsdoc) states:
>   a syntax that is loosely based on JSDoc

The [JSDoc spec](https://jsdoc.app/about-block-inline-tags.html) distinguishes two types of tags: inline and block. Inline uses the `{@tag ...}` syntax and block uses the start of a comment line like:

```ts
/**
 * @blockTag ...
 */
```

The JSDoc spec also states:

>    Each block tag must be followed by a line break, **with the exception of the last block tag in a JSDoc comment**.

There is a `@remarks` block tag that we could use for this. See the [TSDoc playground](https://microsoft.github.io/tsdoc/). This same feedback applies to the other "Note:"s added in this PR.

Reply:
Nice catch! I tried @remarks tag, but that doesn't work. Verified that we use JSII docs to generate the docs website and remarks tag is not supported in JSII docs yet. Hence moving the note to the main comment block for better readability.

Reply:
Great - thanks for investigating that. Resolved